test/src/com/espressif/idf/test/executable/cases/launchconfiguration/LaunchBarCDTConfigurationsTest.java

```diff
@@ -0,0 +1,301 @@
+		fixture.whenProjectIsCreatedFromTemplate();
+		fixture.thenLaunchDescriptorContainsConfig("TestProject");
+		fixture.thenLaunchDescriptorContainsConfig("TestProject2");
+		fixture.selectProjectFromLaunchBar("TestProject");
+		fixture.whenProjectIsBuiltUsingToolbarButton();
+		fixture.thenConsoleShowsBuildSuccessful();
+		fixture.thenConsoleShowsProjectNameInConsole("TestProject");
+		fixture.selectProjectFromLaunchBar("TestProject2");
+		fixture.whenProjectIsBuiltUsingToolbarButton();
+		fixture.thenConsoleShowsBuildSuccessful();
+		fixture.thenConsoleShowsProjectNameInConsole("TestProject2");
+	}
+
+	@Test
+	public void creatingNewGenericLaunchTarget()
+	{
+		fixture.givenNewLaunchTargetIsSelected();
+		fixture.givenTargetTypeSelectedIs("Generic Target");
+		fixture.givenNewLaunchTargetName("TestGenericTarget");
+		fixture.givenNewLauncTargetOperatingSystem("LINUX");
+		fixture.givenNewLaunchTargetCPUArchitecture("x86");
+		fixture.whenFinishIsClicked();
+		fixture.thenLaunchTargetContains("TestGenericTarget");
+		fixture.thenLaunchTargetIsSelectedFromLaunchTargets("TestGenericTarget");
+	}
+
+	@Test
+	public void creatingNewSerialFlashLaunchTarget()
+	{
+		fixture.givenNewLaunchTargetIsSelected();
+		fixture.givenTargetTypeSelectedIs("Serial Flash Target");
+		fixture.givenNewLaunchTargetName("TestSerialFlash");
+		fixture.givenNewLauncTargetOperatingSystem("LINUX");
+		fixture.givenNewLaunchTargetCPUArchitecture("x86");
+		fixture.givenNewLaunchTargetComPortIsSetTo("COMTEST");
+		fixture.whenFinishIsClicked();
+		fixture.thenLaunchTargetContains("TestSerialFlash");
+		fixture.thenLaunchTargetIsSelectedFromLaunchTargets("TestSerialFlash");
+	}
+
+	@Test
+	public void creatingNewEspLaunchTarget()
+	{
+		fixture.givenNewLaunchTargetIsSelected();
+		fixture.givenTargetTypeSelectedIs("ESP Target");
+		fixture.givenNewLaunchTargetName("TestESP");
+		fixture.givenNewLaunchTargetComPortIsSetTo("COMTEST");
+		fixture.givenNewLaunchTargetIdfTargetSetTo("esp32s2");
+		fixture.whenFinishIsClicked();
+		fixture.thenLaunchTargetContains("TestESP");
+		fixture.thenLaunchTargetIsSelectedFromLaunchTargets("TestESP");
+	}
+
+	@Test
+	public void creatingNewGDBLaunchTargetOverSerial()
+	{
+		fixture.givenNewLaunchTargetIsSelected();
+		fixture.givenTargetTypeSelectedIs("GDB Remote Serial");
+		fixture.givenNewLaunchTargeGDBName("TestGDBSerial");
+		fixture.whenFinishIsClicked();
+		fixture.thenLaunchTargetContains("TestGDBSerial");
+		fixture.thenLaunchTargetIsSelectedFromLaunchTargets("TestGDBSerial");
+	}
+
+	@Test
+	public void creatingNewGDBLaunchTargetTCP()
+	{
+		fixture.givenNewLaunchTargetIsSelected();
+		fixture.givenTargetTypeSelectedIs("GDB Remote TCP");
+		fixture.givenNewLaunchTargeGDBTcpName("TestGdbTcp");
+		fixture.givenHostNameOrIpIs("127.0.0.1");
+		fixture.givenPortNumberIs(8087);
+		fixture.whenFinishIsClicked();
+		fixture.thenLaunchTargetContains("TestGdbTcp");
+		fixture.thenLaunchTargetIsSelectedFromLaunchTargets("TestGdbTcp");
+	}
```

Comment:
We need only esp launch target, I think we can remove this as well

Reply:
removed